.github/workflows/docker-image.yml

```diff
@@ -13,32 +13,28 @@ jobs:
     runs-on: ubuntu-latest
     env:
       DOCKER_REPO: jeessy/ddns-go
-      DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
-      DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
       DOCKER_PLATFORMS: linux/amd64,linux/arm,linux/arm64
-      DOCKER_REGISTRY: ""
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
       - name: Set up Docker buildx
-        id: buildx
-        uses: crazy-max/ghaction-docker-buildx@v3
+        uses: docker/setup-buildx-action@v1
       - name: Prepare arguments
-        id: prepare
         run: |
-          DOCKER_TAGS="--tag ${DOCKER_REPO}:edge"
-          if [[ $GITHUB_REF == refs/tags/v* ]]; then
-            DOCKER_TAGS="--tag ${DOCKER_REPO}:latest --tag ${DOCKER_REPO}:${GITHUB_REF#refs/tags/}"
-          fi
-          echo ::set-output name=buildx_args:: --output "type=image,push=true" --platform ${DOCKER_PLATFORMS} ${DOCKER_TAGS} .
```

Comment:
```yaml
on:
  push:
    # Sequence of patterns matched against refs/tags
    tags:
    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
```

could make sure that the GITHUB_REF is started with `refs/tags/v`, so just remove the `if` here.